color/p5js/index.md

```diff
@@ -0,0 +1,13 @@
+---  
+layout: p5js
+author: [Filip Czekajlo]
+author-url: [-]
+title: [colm]
+tags: [P5.js,[Color], [-], ]
```

Comment:
no inner `[]` please only the outer one.  See for example https://raw.githubusercontent.com/fabianmoronzirfas/gestalten-in-code/master/grid/p5js/corridor/index.md